src/Service/JiraApiService.php

```diff
@@ -130,6 +133,38 @@ public function getCurrentUserPermissions(): mixed
         return $this->get(self::API_PATH_MY_PERMISSIONS);
     }
 
+    /**
+     * Get all projects, including archived.
```

Comment:
function comments should be moved to interface unless it is implementation specific